### PR TITLE
Swap VGTS NetRS to NetR9.

### PR DESCRIPTION
**Pull request by staylorofford**
FIRST DATA: 2018-01-24T01:51:00

### CODE REVIEW

**Comment by staylorofford** [comment]
Waiting for confirmation of an antenna swap...

**Comment by staylorofford** [comment]
Antenna swap details added to commit.

**Comment by staylorofford** [comment]
@peanutty I've added you to review for a network team perspective

File: install/sessions.csv [hunk 1]
```diff
@@ -621,7 +621,8 @@ VGTM,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,3,geonet,trimble_netr9 x4,2013-05-14T2
 VGTM,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2016-04-22T02:28:34Z,9999-01-01T00:00:00Z
 VGTR,GeoNet,GNS,TRIMBLE NETRS,GPS,30s,4,geonet,trimble_5700 x4,2005-02-11T00:00:00Z,2016-04-05T23:45:35Z
 VGTR,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2016-04-05T23:45:37Z,9999-01-01T00:00:00Z
-VGTS,GeoNet,GNS,TRIMBLE NETRS,GPS,30s,4,geonet,trimble_5700 x4,2005-03-17T00:00:00Z,9999-01-01T00:00:00Z
+VGTS,GeoNet,GNS,TRIMBLE NETRS,GPS,30s,4,geonet,trimble_5700 x4,2005-03-17T00:00:00Z,2018-01-24T01:50:59Z
+VGTS,GeoNet,GNS,TRIMBLE NETRS,GPS+GLO,30s,0,geonet,trimble netr9 x4,2018-01-24T01:51:00Z,9999-01-01T00:00:00Z
```
Comment (elidana) [inline install/sessions.csv]: please change model type on line 625
TRIMBLE NETRS => TRIMBLE NETR9

**Comment by staylorofford** [comment]
Commit updated with correct entry in the sessions.csv
@elidana look good?

**Pull request merged**